api/controllers/EAppEligibilityQuestionsController.js

```diff
@@ -28,17 +28,7 @@ const EAppEligibilityQuestionsController = {
     },
 
     _fetchUserData(req, res) {
-        const userData = HelperService.getUserData(req, res);
-        EAppEligibilityQuestionsController._checkUserLoggedIn(userData, res);
-
-        return userData;
-    },
-
-    _checkUserLoggedIn(userData, res) {
-        if (!userData.loggedIn) {
-            sails.log.error('User is not logged in', userData);
-            return res.forbidden();
-        }
+        return HelperService.getUserData(req, res);
```

Comment:
I think I may have missed something with my previous comment.  It looks like it needs to use the user data to check if they're logged in so this probably needs to be reinstated as this validation check is now removed with your latest change

Reply:
Yes you're correct, it does use the user_data to check if someone is logged in or not but your original suggestion for refactoring still valid since I'm disabling the need for a user to be logged in to view the pages.

Sorry I should have made that clearer in the description 🤦

Reply:
As discussed, the requirement to be logged in is no longer needed for this section, hence, it's been removed.